src/targets/gpu/target.cpp

```diff
@@ -93,10 +93,10 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         dead_code_elimination{},
         eliminate_common_subexpression{},
         dead_code_elimination{},
+        auto_contiguous{},
         simplify_algebra{},
         simplify_reshapes{},
         simplify_algebra{},
-        auto_contiguous{},
```

Comment:
Why is this moved? I think contiguous could hinder some optimizations.

Reply:
I moved the adding contiguous to parse onnx. Reverted this change.